src/controllers/destination.ts

```diff
@@ -169,15 +166,15 @@ export class DestinationController {
   }
 
   public static batchProcess(ctx: Context) {
-    logger.debug(
-      'Native(Process-Transform-Batch):: Requst to transformer::',
-      JSON.stringify(ctx.request.body),
-    );
+    logger.info('Native(Process-Transform-Batch):: Requst to transformer::', ctx.request.body);
     const startTime = new Date();
     const requestMetadata = MiscService.getRequestMetadata(ctx);
     const routerRequest = ctx.request.body as RouterTransformationRequest;
     const destination = routerRequest.destType;
     let events = routerRequest.input;
+    const loggerWithCtx = logger.child({
+      ...MiscService.getLoggableData(events[0]?.metadata as unknown as ErrorDetailer),
+    });
```

Comment:
```suggestion
     const logMetadata = MiscService.getLoggableData(events[0]?.metadata as unknown as ErrorDetailer)
    const loggerWithCtx = logger.child(logMetadata);
```
Do we really need `as unknown` ?

Reply:
to map to ErrorDetailer, ts won't allow converting metadata before we convert to unkown. To get away with linting error used this